.gitignore

```diff
@@ -0,0 +1,12 @@
+__pycache__/
+*.py[cod]
```

Comment:
Please add `bench-results.json`.
You can see it's usage in the makefile under `bench` section.